plan/planbuilder.go

```diff
@@ -325,6 +329,88 @@ func (b *planBuilder) buildAdmin(as *ast.AdminStmt) Plan {
 	return p
 }
 
+func detachIndexColumns(tn *ast.TableName) (indexOffset []int, columnOffset []int) {
```

Comment:
Why only build index columns? Normal columns needs histogram too.
We need histograms for each column, and for each multi-column index.
For `create talbe (a int, b int, c int, index (a, b), index (c))`
We need histogram for column `a`, column `b`, column `c` and index `(a, b)`, 4 histograms in total.